cpu/kinetis_common/spi.c

```diff
@@ -478,7 +490,7 @@ static int find_closest_baudrate_scalers(const uint32_t module_clock, const uint
     };
     static const int br_prescalers[4] = {2, 3, 5, 7};
 
-    int closest_frequency = -1;
+    long closest_frequency = -1;
```

Comment:
This is your return value. Now you are returning a long even if the return type is int. Doesn't this cause warnings?